assimilation_code/modules/assimilation/probit_transform_mod.f90

```diff
@@ -0,0 +1,733 @@
+elseif(p%distribution_type == UNIFORM_DISTRIBUTION) then 
+   call to_probit_uniform(ens_size, state_ens, p, probit_ens, use_input_p, lower_bound, upper_bound)
+elseif(p%distribution_type == GAMMA_DISTRIBUTION) then 
+   call to_probit_gamma(ens_size, state_ens, p, probit_ens, use_input_p, &
+      bounded_below, bounded_above, lower_bound, upper_bound)
+elseif(p%distribution_type == BETA_DISTRIBUTION) then 
+   call to_probit_beta(ens_size, state_ens, p, probit_ens, use_input_p, &
+      lower_bound, upper_bound)
+elseif(p%distribution_type == BOUNDED_NORMAL_RH_DISTRIBUTION) then
+   call to_probit_bounded_normal_rh(ens_size, state_ens, p, probit_ens, &
+      use_input_p, bounded_below, bounded_above, lower_bound, upper_bound)
+
+!----------------------------------------------------------------------------------
+! The following code block tests that the to/from probit calls are nearly inverse
+! for all of the calls made during an assimilation
+   if(do_inverse_check) then
+      if(.not. use_input_p) then
+         call to_probit_bounded_normal_rh(ens_size, state_ens, p_temp, probit_ens_temp, &
+            use_input_p, bounded_below, bounded_above, lower_bound, upper_bound)
+         call from_probit_bounded_normal_rh(ens_size, probit_ens_temp, p_temp, state_ens_temp)
+         diff = state_ens - state_ens_temp
+         if(abs(maxval(diff)) > 1.0e-8_r8) then
+            write(*, *) 'Maximum allowed value of probit to/from difference exceeded'
+            write(*, *) 'Location of minimum ensemble member ', minloc(state_ens)
+            write(*, *) 'Location of maximum ensemble member ', maxloc(state_ens)
+            do i = 1, ens_size
+               write(*, *) i, state_ens(i), state_ens_temp(i), diff(i)
+            enddo
+            stop
+         endif
+      endif
+   
+      if(use_input_p) then
+         call to_probit_bounded_normal_rh(ens_size, state_ens, p, probit_ens_temp, &
+            use_input_p, bounded_below, bounded_above, lower_bound, upper_bound)
+         call from_probit_bounded_normal_rh(ens_size, probit_ens_temp, p, state_ens_temp)
+         diff = state_ens - state_ens_temp
+         if(abs(maxval(diff)) > 1.0e-8_r8) then
+            write(*, *) 'Maximum allowed value of probit to/from difference for input p exceeded'
+            write(*, *) 'Location of minimum ensemble member ', minloc(state_ens)
+            write(*, *) 'Location of maximum ensemble member ', maxloc(state_ens)
+            do i = 1, ens_size
+               write(*, *) i, state_ens(i), state_ens_temp(i), diff(i)
+            enddo
+            stop
+         endif
+      
+      endif
+   endif
+!----------------------------------------------------------------------------------
+
+
+!!!elseif(p%distribution_type == PARTICLE_FILTER_DISTRIBUTION) then
+   !!!call to_probit_particle(ens_size, state_ens, p, probit_ens, use_input_p, &
+       !!!bounded_below, bounded_above, lower_bound, upper_bound)
```

Comment:
Are we still planning on supporting `distribution_type = PARTICLE_FILTER_DISTRIBUTION ` in the future?

Reply:
remove PARTICLE_FILTER_DISTRIBUTION from list of available distributions 

https://github.com/NCAR/DART/blob/d5c21646ebef17865cfe779fd68894cbe6a51112/guide/qceff_probit.rst?plain=1#L123-L132

since it is not available in this code.

Reply:
@mjs2369 I'm not sure when PARTICLE_FILTER_DISTRIBUTION will be supported, 
I believe @jlaucar has experimented/tested with this. 
For now I fixed the docs to match the code. Maybe PARTICLE_FILTER_DISTRIBUTION  goes in v11.1?

Reply:
Yep I agree that the PARTICLE_FILTER_DISTRIBUTION can be put into a later release if we want. Good catch with removing it from this release of the documentation. 

Should we also remove it from this comment in algorithm_info_mod?
[https://github.com/NCAR/DART/blob/9e6b2068602d302aae4b5d12480b7a8dd2bade1b/assimilation_code/modules/assimilation/algorithm_info_mod.f90#L386](https://github.com/NCAR/DART/blob/9e6b2068602d302aae4b5d12480b7a8dd2bade1b/assimilation_code/modules/assimilation/algorithm_info_mod.f90#L386)

Reply:
@mjs2369 it is not just the comment right? It is the code in algortihm_info_mod also:
https://github.com/NCAR/DART/blob/9e6b2068602d302aae4b5d12480b7a8dd2bade1b/assimilation_code/modules/assimilation/algorithm_info_mod.f90#L249-L250

@jlaucar do you want particle_filter_distritbution in this release?

Reply:
Yes we should remove that as well. I feel like we the code would potentially take a while to be ready for this release and we should probably put it in with a later one

If we do as such, should we also remove the commented out code pertaining to it in probit_transform_mod? Lines 187-9 and the subroutine to_probit_particle starting on line 378

Reply:
@mjs2369 chatted with Jeff about this. 
Conclusion: don't let people select PARTICLE_FILTER_DISTRIBUTION.  Leave in the commented subroutine to_probit_particle (if we were being strict about code we would remove this, but this is science).  I'll go ahead and remove `case ('PARTICLE_FILTER_DISTRIBUTION') ` algorithm_info_mod.f90.   Once again thanks for the detailed review on this. 

Cheers,
Helen